spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'projects show ', type: :feature do
+  describe 'as a user' do
+    let(:project) do
+      recycled_material_challenge = Challenge.create!(theme: 'Recycled Material', project_budget: 1000)
+      recycled_material_challenge.projects.create!(name: 'News Chic', material: 'Newspaper')
+    end
+
+    scenario 'when I visit a project show page, I should see' do
+      visit "/projects/#{project.id}"
+
+      expect(page).to have_content(project.name)
+      expect(page).to have_content(project.material)
+      expect(page).to have_content(project.challenge.theme)
+      expect(page).to have_content(project.contestants.count)
+    end
```

Comment:
I'd love to see more specificity with your tests. The project name could be anywhere on the page, so use of labels (ex: `"Project Name: #{project.name}"`) is helpful or within blocks.